### ISSUE TITLE
Basin related machine unable to continously working

### ISSUE THREAD

**Issue by kxdxzr** [Fabricators-of-Create/Create#1657]
### Description

Hi Create Fabric Member,

Game version: 1.20.1
Launcher: HMCL V3.5.9
Mod version: create-fabric-0.5.1-j-build.1604+mc1.20.1

![image](https://github.com/user-attachments/assets/86542992-6de6-4768-9251-db1cb2147dc1)
Figure 1. Content in the Basin

![image](https://github.com/user-attachments/assets/c139c702-1a9c-4b29-b4be-ddcb357b8111)
Figure 2. Power supply to the mixer

![image](https://github.com/user-attachments/assets/24a68edc-57ca-4844-bd4a-ea4280ea6f76)
Figure 3. The press in another machine doesn't work as well.

For the machine containing **Basin**, it seems that the machine only works for a few times and stops working. It will work for a few times if I re-place the basin and **stop**  again. I found this issue in machines with mixer and press. Both of them are on the belt and their input ports are all through Funnels.  I guess it is related with some block updating?

While it is hard for me to provide a schematic for all the machines I mentioned as I made those things directly in survival without proper routing but I can provide the save of the game if necessary.

I reported this on Discord as well and someone mentioned that he tried and it does not work in 1.19 as well.

Sorry for any unclear in specific words, I used the following website for translation reference: https://www.mcmod.cn/item/list/2021-1.html

Best reagrads,
Kxdxzr

### Game Log

https://api.mclo.gs/1/raw/8rRDTcJ

### Debug Information

<details>
<summary>Client Info</summary>

```
Create:
	Mod Version: 0.5.1j
	Fabric API Version: 0.92.2+1.20.1
	Minecraft Version: 1.20.1

Graphics:
	Flywheel Version: 0.6.11-4
	Flywheel Backend: INSTANCING
	OpenGL Renderer: Intel(R) UHD Graphics
	OpenGL Version: 3.2.0 - Build 31.0.101.2130
	Graphics Mode: options.graphics.fancy

System Information:
	Operating System: Windows 11 (amd64) version 10.0
	Java Version: 21.0.3, BellSoft
	JVM Flags: 11 total; -Xmx4096m -XX:+UnlockExperimentalVMOptions -XX:+UseG1GC -XX:G1NewSizePercent=20 -XX:G1ReservePercent=20 -XX:MaxGCPauseMillis=50 -XX:G1HeapRegionSize=32m -XX:-UseAdaptiveSizePolicy -XX:-OmitStackTraceInFastThrow -XX:-DontCompileHugeMethods -XX:HeapDumpPath=MojangTricksIntelDriversForPerformance_javaw.exe_minecraft.exe.heapdump
	Memory: 801580104 bytes (764 MiB) / 2583691264 bytes (2464 MiB) up to 4294967296 bytes (4096 MiB)
	Total Memory: 15552135168 bytes (14825 MiB) / 16943685632 bytes (16152 MiB)
	CPU: Intel(R) Core(TM) i7-10870H CPU @ 2.20GHz @ 2.21 GHz; 8 cores / 16 threads on 1 socket(s)
	Graphics card #0: Intel(R) UHD Graphics (Intel Corporation (0x8086)); 1024.00 MB of VRAM
	Graphics card #1: NVIDIA GeForce RTX 3060 Laptop GPU (NVIDIA (0x10de)); 4095.00 MB of VRAM

Other Mods:
	Cardinal Components API (base): 5.2.0
	Cardinal Components API (entities): 5.2.0
	core: 3.6.6
	toml: 3.6.6
	jsr305: 3.0.2
	Dripstone Fluid Lib: 3.0.2
	Fabric API Base: 0.4.31+1802ada577
	Fabric API Lookup API (v1): 1.6.36+1802ada577
	Fabric Biome API (v1): 13.0.13+1802ada577
	Fabric Block API (v1): 1.0.11+1802ada577
	Fabric BlockView API (v2): 1.0.1+1802ada577
	Fabric BlockRenderLayer Registration (v1): 1.1.41+1802ada577
	Fabric Client Tags: 1.1.2+1802ada577
	Fabric Command API (v1): 1.2.34+f71b366f77
	Fabric Command API (v2): 2.2.13+1802ada577
	Fabric Commands (v0): 0.2.51+df3654b377
	Fabric Containers (v0): 0.1.64+df3654b377
	Fabric Content Registries (v0): 4.0.11+1802ada577
	Fabric Convention Tags: 1.5.5+1802ada577
	Fabric Crash Report Info (v1): 0.2.19+1802ada577
	Fabric Data Attachment API (v1): 1.0.0+de0fd6d177
	Fabric Data Generation API (v1): 12.3.4+1802ada577
	Fabric Dimensions API (v1): 2.1.54+1802ada577
	Fabric Entity Events (v1): 1.6.0+1c78457f77
	Fabric Events Interaction (v0): 0.6.2+1802ada577
	Fabric Events Lifecycle (v0): 0.2.63+df3654b377
	Fabric Game Rule API (v1): 1.0.40+1802ada577
	Fabric Item API (v1): 2.1.28+1802ada577
	Fabric Item Group API (v1): 4.0.12+1802ada577
	Fabric Key Binding API (v1): 1.0.37+1802ada577
	Fabric Key Bindings (v0): 0.2.35+df3654b377
	Fabric Language Kotlin: 1.13.0+kotlin.2.1.0
	Fabric Lifecycle Events (v1): 2.2.22+1802ada577
	Fabric Loot API (v2): 1.2.1+1802ada577
	Fabric Loot Tables (v1): 1.1.45+9e7660c677
	Fabric Message API (v1): 5.1.9+1802ada577
	Fabric Mining Level API (v1): 2.1.50+1802ada577
	Fabric Model Loading API (v1): 1.0.3+1802ada577
	Fabric Models (v0): 0.4.2+9386d8a777
	Fabric Networking API (v1): 1.3.11+1802ada577
	Fabric Networking (v0): 0.3.51+df3654b377
	Fabric Object Builder API (v1): 11.1.3+1802ada577
	Fabric Particles (v1): 1.1.2+1802ada577
	Fabric Recipe API (v1): 1.0.21+1802ada577
	Fabric Registry Sync (v0): 2.3.3+1802ada577
	Fabric Renderer API (v1): 3.2.1+1802ada577
	Fabric Renderer - Indigo: 1.5.2+85287f9f77
	Fabric Renderer Registries (v1): 3.2.46+df3654b377
	Fabric Rendering Data Attachment (v1): 0.3.37+92a0d36777
	Fabric Rendering Fluids (v1): 3.0.28+1802ada577
	Fabric Rendering (v0): 1.1.49+df3654b377
	Fabric Rendering (v1): 3.0.8+1802ada577
	Fabric Resource Conditions API (v1): 2.3.8+1802ada577
	Fabric Resource Loader (v0): 0.11.10+1802ada577
	Fabric Screen API (v1): 2.0.8+1802ada577
	Fabric Screen Handler API (v1): 1.3.30+1802ada577
	Fabric Sound API (v1): 1.0.13+1802ada577
	Fabric Transfer API (v1): 3.3.5+8dd72ea377
	Fabric Transitive Access Wideners (v1): 4.3.1+1802ada577
	Fabric Loader: 0.16.9
	Forge Config API Port: 8.0.0
	Inventory Profiles Next: 1.10.11
	OpenJDK 64-Bit Server VM: 21
	Just Enough Items: 15.20.0.105
	libIPN: 4.0.2
	Litematica: 0.15.4
	MaLiLib: 0.16.3
	Milk Lib: 1.2.60
	MiniHUD: 0.27.1
	MixinExtras: 0.4.1
	Mod Menu: 7.2.2
	kotlin-reflect: 2.1.0
	kotlin-stdlib: 2.1.0
	kotlin-stdlib-jdk7: 2.1.0
	kotlin-stdlib-jdk8: 2.1.0
	atomicfu-jvm: 0.26.1
	kotlinx-coroutines-core-jvm: 1.9.0
	kotlinx-coroutines-jdk8: 1.9.0
	kotlinx-datetime-jvm: 0.6.1
	kotlinx-io-bytestring-jvm: 0.6.0
	kotlinx-io-core-jvm: 0.6.0
	kotlinx-serialization-cbor-jvm: 1.7.3
	kotlinx-serialization-core-jvm: 1.7.3
	kotlinx-serialization-json-jvm: 1.7.3
	Porting Lib Accessors: 2.3.8+1.20.1
	Porting Lib Attributes: 2.3.8+1.20.1
	Porting Lib Base: 2.3.8+1.20.1
	Porting Lib Brewing: 2.3.8+1.20.1
	Porting Lib Client Events: 2.3.8+1.20.1
	Porting Lib Common: 2.3.8+1.20.1
	Porting Lib Config: 2.3.8+1.20.1
	Porting Lib Core: 2.3.8+1.20.1
	Porting Lib Data: 2.1.1090+1.20
	Porting Lib Entity: 2.3.8+1.20.1
	Porting Lib Extensions: 2.3.8+1.20.1
	Porting Lib Fluids: 2.3.8+1.20.1
	Porting Lib GameTest: 2.1.1090+1.20
	Porting Lib Gui Utils: 2.3.8+1.20.1
	Porting Lib Lazy Register: 2.3.8+1.20.1
	Porting Lib Mixin Extensions: 2.3.8+1.20.1
	Porting Lib Model Generators: 2.1.1090+1.20
	Porting Lib Model Loader: 2.3.8+1.20.1
	Porting Lib Model Materials: 2.1.1090+1.20
	Porting Lib Models: 2.3.8+1.20.1
	Porting Lib Networking: 2.3.8+1.20.1
	Porting Lib Obj Loader: 2.3.8+1.20.1
	Porting Lib Tags: 3.0
	Porting Lib Tool Actions: 2.3.8+1.20.1
	Porting Lib Transfer: 2.3.8+1.20.1
	Porting Lib Utility: 2.3.8+1.20.1
	Reach Entity Attributes: 2.4.0
	Registrate for Fabric: 1.3.79-MC1.20.1
	Trinkets: 3.7.2
	Tweakeroo: 0.17.1
```

</details>

<details>
<summary>Server Info</summary>

```
Create:
	Mod Version: 0.5.1j
	Fabric API Version: 0.92.2+1.20.1
	Minecraft Version: 1.20.1

System Information:
	Operating System: Windows 11 (amd64) version 10.0
	Java Version: 21.0.3, BellSoft
	JVM Flags: 11 total; -Xmx4096m -XX:+UnlockExperimentalVMOptions -XX:+UseG1GC -XX:G1NewSizePercent=20 -XX:G1ReservePercent=20 -XX:MaxGCPauseMillis=50 -XX:G1HeapRegionSize=32m -XX:-UseAdaptiveSizePolicy -XX:-OmitStackTraceInFastThrow -XX:-DontCompileHugeMethods -XX:HeapDumpPath=MojangTricksIntelDriversForPerformance_javaw.exe_minecraft.exe.heapdump
	Memory: 801580104 bytes (764 MiB) / 2583691264 bytes (2464 MiB) up to 4294967296 bytes (4096 MiB)
	Total Memory: 15552135168 bytes (14825 MiB) / 16943685632 bytes (16152 MiB)
	CPU: Intel(R) Core(TM) i7-10870H CPU @ 2.20GHz @ 2.21 GHz; 8 cores / 16 threads on 1 socket(s)
	Graphics card #0: Intel(R) UHD Graphics (Intel Corporation (0x8086)); 1024.00 MB of VRAM
	Graphics card #1: NVIDIA GeForce RTX 3060 Laptop GPU (NVIDIA (0x10de)); 4095.00 MB of VRAM

Other Mods:
	Cardinal Components API (base): 5.2.0
	Cardinal Components API (entities): 5.2.0
	core: 3.6.6
	toml: 3.6.6
	jsr305: 3.0.2
	Dripstone Fluid Lib: 3.0.2
	Fabric API Base: 0.4.31+1802ada577
	Fabric API Lookup API (v1): 1.6.36+1802ada577
	Fabric Biome API (v1): 13.0.13+1802ada577
	Fabric Block API (v1): 1.0.11+1802ada577
	Fabric BlockView API (v2): 1.0.1+1802ada577
	Fabric BlockRenderLayer Registration (v1): 1.1.41+1802ada577
	Fabric Client Tags: 1.1.2+1802ada577
	Fabric Command API (v1): 1.2.34+f71b366f77
	Fabric Command API (v2): 2.2.13+1802ada577
	Fabric Commands (v0): 0.2.51+df3654b377
	Fabric Containers (v0): 0.1.64+df3654b377
	Fabric Content Registries (v0): 4.0.11+1802ada577
	Fabric Convention Tags: 1.5.5+1802ada577
	Fabric Crash Report Info (v1): 0.2.19+1802ada577
	Fabric Data Attachment API (v1): 1.0.0+de0fd6d177
	Fabric Data Generation API (v1): 12.3.4+1802ada577
	Fabric Dimensions API (v1): 2.1.54+1802ada577
	Fabric Entity Events (v1): 1.6.0+1c78457f77
	Fabric Events Interaction (v0): 0.6.2+1802ada577
	Fabric Events Lifecycle (v0): 0.2.63+df3654b377
	Fabric Game Rule API (v1): 1.0.40+1802ada577
	Fabric Item API (v1): 2.1.28+1802ada577
	Fabric Item Group API (v1): 4.0.12+1802ada577
	Fabric Key Binding API (v1): 1.0.37+1802ada577
	Fabric Key Bindings (v0): 0.2.35+df3654b377
	Fabric Language Kotlin: 1.13.0+kotlin.2.1.0
	Fabric Lifecycle Events (v1): 2.2.22+1802ada577
	Fabric Loot API (v2): 1.2.1+1802ada577
	Fabric Loot Tables (v1): 1.1.45+9e7660c677
	Fabric Message API (v1): 5.1.9+1802ada577
	Fabric Mining Level API (v1): 2.1.50+1802ada577
	Fabric Model Loading API (v1): 1.0.3+1802ada577
	Fabric Models (v0): 0.4.2+9386d8a777
	Fabric Networking API (v1): 1.3.11+1802ada577
	Fabric Networking (v0): 0.3.51+df3654b377
	Fabric Object Builder API (v1): 11.1.3+1802ada577
	Fabric Particles (v1): 1.1.2+1802ada577
	Fabric Recipe API (v1): 1.0.21+1802ada577
	Fabric Registry Sync (v0): 2.3.3+1802ada577
	Fabric Renderer API (v1): 3.2.1+1802ada577
	Fabric Renderer - Indigo: 1.5.2+85287f9f77
	Fabric Renderer Registries (v1): 3.2.46+df3654b377
	Fabric Rendering Data Attachment (v1): 0.3.37+92a0d36777
	Fabric Rendering Fluids (v1): 3.0.28+1802ada577
	Fabric Rendering (v0): 1.1.49+df3654b377
	Fabric Rendering (v1): 3.0.8+1802ada577
	Fabric Resource Conditions API (v1): 2.3.8+1802ada577
	Fabric Resource Loader (v0): 0.11.10+1802ada577
	Fabric Screen API (v1): 2.0.8+1802ada577
	Fabric Screen Handler API (v1): 1.3.30+1802ada577
	Fabric Sound API (v1): 1.0.13+1802ada577
	Fabric Transfer API (v1): 3.3.5+8dd72ea377
	Fabric Transitive Access Wideners (v1): 4.3.1+1802ada577
	Fabric Loader: 0.16.9
	Forge Config API Port: 8.0.0
	Inventory Profiles Next: 1.10.11
	OpenJDK 64-Bit Server VM: 21
	Just Enough Items: 15.20.0.105
	libIPN: 4.0.2
	Litematica: 0.15.4
	MaLiLib: 0.16.3
	Milk Lib: 1.2.60
	MiniHUD: 0.27.1
	MixinExtras: 0.4.1
	Mod Menu: 7.2.2
	kotlin-reflect: 2.1.0
	kotlin-stdlib: 2.1.0
	kotlin-stdlib-jdk7: 2.1.0
	kotlin-stdlib-jdk8: 2.1.0
	atomicfu-jvm: 0.26.1
	kotlinx-coroutines-core-jvm: 1.9.0
	kotlinx-coroutines-jdk8: 1.9.0
	kotlinx-datetime-jvm: 0.6.1
	kotlinx-io-bytestring-jvm: 0.6.0
	kotlinx-io-core-jvm: 0.6.0
	kotlinx-serialization-cbor-jvm: 1.7.3
	kotlinx-serialization-core-jvm: 1.7.3
	kotlinx-serialization-json-jvm: 1.7.3
	Porting Lib Accessors: 2.3.8+1.20.1
	Porting Lib Attributes: 2.3.8+1.20.1
	Porting Lib Base: 2.3.8+1.20.1
	Porting Lib Brewing: 2.3.8+1.20.1
	Porting Lib Client Events: 2.3.8+1.20.1
	Porting Lib Common: 2.3.8+1.20.1
	Porting Lib Config: 2.3.8+1.20.1
	Porting Lib Core: 2.3.8+1.20.1
	Porting Lib Data: 2.1.1090+1.20
	Porting Lib Entity: 2.3.8+1.20.1
	Porting Lib Extensions: 2.3.8+1.20.1
	Porting Lib Fluids: 2.3.8+1.20.1
	Porting Lib GameTest: 2.1.1090+1.20
	Porting Lib Gui Utils: 2.3.8+1.20.1
	Porting Lib Lazy Register: 2.3.8+1.20.1
	Porting Lib Mixin Extensions: 2.3.8+1.20.1
	Porting Lib Model Generators: 2.1.1090+1.20
	Porting Lib Model Loader: 2.3.8+1.20.1
	Porting Lib Model Materials: 2.1.1090+1.20
	Porting Lib Models: 2.3.8+1.20.1
	Porting Lib Networking: 2.3.8+1.20.1
	Porting Lib Obj Loader: 2.3.8+1.20.1
	Porting Lib Tags: 3.0
	Porting Lib Tool Actions: 2.3.8+1.20.1
	Porting Lib Transfer: 2.3.8+1.20.1
	Porting Lib Utility: 2.3.8+1.20.1
	Reach Entity Attributes: 2.4.0
	Registrate for Fabric: 1.3.79-MC1.20.1
	Trinkets: 3.7.2
	Tweakeroo: 0.17.1
```

</details>

**Comment by gjunnila** [comment]
I can replicate this on the same build. Funnel outputting to a belt no longer works continuously whereas on the previous build it did.
Breaking and placing the funnel fixes it for a single craft.

**Comment by anmvc** [comment]
Presses nor the Basin work for me either, it looks like it might be the same issue as it's getting power and using the stress units but doing nothing

**Comment by kxdxzr** [comment]
Hi Create Fabric Member,

One follow-up information.

![image](https://github.com/user-attachments/assets/ef0e53b2-fc3e-467d-925f-a1d76fcf587b)
Figure 1. Press with a different IO configuration.

The structure in figure 1 seems to have a more rare possibility for this bug.

Best regards,
kxdxzr

**Comment by BugInYourCode01** [comment]
Thank you very much. Was it a problem with the Basins output? I(I just found out that basins cannot emty themselves into other inventories and therefore the mixers stopped)

Even though this might be "version spamming", please release a third patch for this problem, because the bug broke almost EVERY factory.

**Comment by IThundxr** [comment]
This has been fixed by 0.5.1-i patch 2 on 1.18.2, 0.5.1-i patch 3 on 1.19.2 and 0.5.1-j patch 3 on 1.20.1
